python/lsst/ts/scheduler/driver/observation.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of ts_scheduler
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Vera C. Rubin Observatory instead of LSST please. And check all other files as well.